### PR TITLE
Poolmgr concurrency

### PR DESCRIPTION
**Pull request by therahulbhati**

### CODE REVIEW

File: docs/content/en/docs/concepts/components/core/executor.md [hunk 1]
```diff
@@ -56,9 +58,10 @@ then a new pod is specialised from the pool and used for execution.
 PoolManager is great for functions that are **short-living** and requires a **short 
 cold start time** [1].
 
-However, PoolManager has certain limitations. It selects only one pod per function, 
-which is not suitable for serving massive traffic. In such cases, you should consider
-using NewDeploy as executor type of function.
+In previous versions, PoolManager has certain limitations. It used to select only 
+one pod per function, which is not suitable for serving massive traffic. To overcome
```
Comment (vishal-biyani) [inline docs/content/en/docs/concepts/components/core/executor.md:62]:
```suggestion
one pod per function, which is not suitable if you want to serve more requests in parallel. To overcome 
```

File: docs/content/en/docs/usage/executor.en.md [hunk 2]
```diff
@@ -9,11 +9,12 @@ In this guide, we will go through how to set up/config different executors for f
 
 ### Poolmgr (Pool-based executor)
 
-You can create a function like following to use poolmgr as backend executor.
+You can create a function like following to use poolmgr as backend executor. You can use the `concurrency` field to control the 
+maximum number of concurrent pod specialization(default 5) to serve requests. As of now, `concurrency` field is only supported by poolmgr.
```
Comment (vishal-biyani) [inline docs/content/en/docs/usage/executor.en.md:13]:
```suggestion
maximum number of concurrent pod specialization(default 5) to serve requests. The `concurrency` field is only supported by poolmgr.
```

**Review by vishal-biyani — CHANGES_REQUESTED** [review]

**Comment by vishal-biyani** [comment]
@therahulbhati Resolve Conflicts please.

**Review by pyrkamarcin — APPROVED** [review]

**Comment by sanketsudake** [comment]
Changes ported to PR https://github.com/fission/docs.fission.io/pull/296

**Pull request closed without merging**